src/webui.c

```diff
@@ -174,6 +195,17 @@ process_request(struct evhttp_request *req, void *arg)
         send_json_workers(req, arg);
         return;
     }
+    
+    char *worker_name_start = strstr(url, "/worker/");
+    if ( worker_name_start != NULL)
+    {
+        char rig_id[MAX_RIG_ID] = {0};
+        worker_name_start += 8;
+        char *decoded_worker_name = evhttp_decode_uri(worker_name_start);
```

Comment:
evhttp_uridecode, you need to pass the 1 for decoding plus.

Reply:
eta sun apr 18/2021